src/ChatkitGiftedProvider.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+const { ChatkitProvider } = require('@pusher/chatkit-client-react');
+import { ActiveRoomsProvider } from './ActiveRoomsProvider';
+import { ChatRoomProvider } from './ChatRoomProvider';
+
+interface Props {
+  userId: string,
+  instanceLocator: string,
+  tokenProvider: () => string,
+  children: React.ReactChild,
+}
+
+export const ChatkitGiftedProvider = ({ userId, instanceLocator, tokenProvider, children }: Props) => {
+  const userIdIsString = typeof userId === 'string';
+  if (!userId || !userIdIsString) {
+    throw new Error('Please ensure you set userId to a string otherwise chat won\'t work!');
```

Comment:
You could possibly just always do `String(userId)` or `userId.toString()`

Reply:
good idea